setup.py

```diff
@@ -104,6 +104,7 @@ def get_git_sha():
         "simplejson>=3.15.0",
         "sqlalchemy>=1.3.5, <2.0",
         "sqlalchemy-utils>=0.33.2",
+	"snowflake-sqlalchemy>=1.2.0",
```

Comment:
Please fix the indentation here and move to `extras_require`; `snowflake-sqlachemy` is not required to run Superset.